modules/payments/backend/modules/payments/viewsets.py

```diff
@@ -60,3 +69,36 @@ def get(self, request, *args, **kwargs):
             "data": history
         }
         return Response(response, status=status.HTTP_200_OK)
+
+class AppleIAProductsView(APIView):
+    authentication_classes = [authentication.TokenAuthentication]
+    permission_classes = [permissions.IsAuthenticated]
+
+    def get(self, request, *args, **kwargs):
+        products = AppleIAPProduct.objects.filter(is_active=True)
+        response = {
+            "success": True,
+            "data": [obj.as_dict() for obj in products]
+        }
+        return Response(response, status=status.HTTP_200_OK)
```

Comment:
Why not use as ListView with queryset filtered to is_active and a serializer of AppleIAPProduct? https://www.django-rest-framework.org/tutorial/3-class-based-views/#using-generic-class-based-views